Fix code snippets (#585)
vtex-apps/io-documentation: docs/en/Recipes/templates/customizing-the-header-and-footer-blocks-by-page.md

In the desired template, declare the code given as example below to overwrite the Header and Footer automatic duplication.  

> :information_source: *Remember to change the CHANGEME to a valid page name, example: "product", "search#category", "custom", etc*

Reviewer: ```suggestion
<div class="alert alert-info">
Remember to replace the <code>{headerBlock}</code> and <code>{footerBlock}</code> values with real block names. Also, replace the <code>{templateName}</code> value with a valid theme template, such as <code>product</code>, <code>search#category</code> and <code>custom</code>. 
</div>

```

```json
{

>   "store.CHANGEME": {

Reviewer: ```suggestion
  "store.{templateName}": {
```

    "parent": { 

>       "header": "header#other-header", 
>       "footer": "footer#other-footer"

Reviewer: ```suggestion
      "header": "{headerBlock}", 
      "footer": "{footerBlock}"
```

    }
  }
}
```

The code above works in scenarios where the Header **and** Footer will be overwritten. When overwriting just one of the two, keep the template's `parent` structure and determine which block will be customized. For example:

```json
{

>   "store.CHANGEME": {

Reviewer: ```suggestion
  "store.{templateName}": {
```

    "parent": { 

>       "header": "header#other-header"

Reviewer: ```suggestion
      "header": "{headerBlock}"
```
